server/Middleware/auth.js

```diff
@@ -0,0 +1,21 @@
+const jwt = require("jsonwebtoken");
+
+const config = process.env;
+
+const verifyToken = (req, res, next) => {

```

Comment:
Any thoughts on using external authentication services?

Reply:
Google auth can be an option

Reply:
What have you been using before @mutebironald ?

Reply:
I have used social Auth on the Front end like Google, facebook